src/main/java/com/deloitte/elrr/services/config/AuditorAwareImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.deloitte.elrr.services.config;
+
+import java.util.Optional;
+
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+/**
+ * Implementation of AuditorAware to provide the current auditor for JPA
+ * auditing. This automatically sets the updatedBy field in auditable entities.
+ */
+@Component
+public class AuditorAwareImpl implements AuditorAware<String> {
```

Comment:
oh so this is something that the per-table field wants to know?

Reply:
not sure what you mean here